source/includes/_evaluation-api.md

```diff
@@ -0,0 +1,37 @@
+# Evaluation API
+```shell
+curl -X POST https://collection-prod.inspcdn.net/evaluation \
+    -H "Authorization: Bearer $API_KEY" \
+    -H "Content-Type: application/json" \
+    -d '{
+        "sale_id": "12345",
+        "account_id": "157421",
+        "sale_datetime": 1579792758,
+        "event_date_id": "23553",
+        "sale_total_value": 54.26,
+        "first_six_digits_cc": "455326",
+        "last_four_digits_cc": "0012",
+        "holder_cpf": "741.112.235-53"
+    }'
+```
+
+Our evaluation API consists of a single resource: `/evaluation`. This resource should get used whenever you have a Sale that you would like to get evaluated by Inspetor.
+
+It is a `POST` endpoint, and the fields it accepts are as follows:
+
+Property                        | Type    | Description
+--------                        | ----    | -----------
+sale_id                         | String  | The sale ID
+account_id                      | String  | The ID of the account attempting the purchase
+sale_datetime                   | Integer | The Unix-timestamp corresponding to the date and time of the sale
+event_date_id <sup>[*]</sup>    | String  | The event datetime id associated to the sale
+sale_total_value                | Float   | The total monetary value of the sale
+first_six_digits_cc             | String  | The first six digits of the credit card associated to the sale
+last_four_digits_cc             | String  | The last four digits of the credit card associated to the sale
+holder_cpf                      | String  | The card holder's CPF. It may contain dots and dashes but it's not required.
+
+All of these fields are required - we cannot evaluate a sale that has any of these missing.
+
+[*] Even though a sale might have more than one event datetime id associated to it, at this moment, we only require the main one associated to the sale in this endpoint. That is, even if there might be multiple tickets, corresponding to multiple event datetime ids, in a single sale, we only ask for a single event datetime id.
```

Comment:
Do we expect them to send the `event_date_id` for the closest event_date?

Reply:
no, it's the one in their sale table

Reply:
Maybe we should aside here to make it more visible. bundle exec middleman server

Reply:
> no, it's the one in their sale table

Ok!

Reply:
Do you guys think it's weird if I add a very Ingresse-specific comment like "(usually the event_date_id that will be in your sales table)"?

Reply:
A little bit, but we do have `ingresse` as a possible `fraud_signal_type`

Reply:
(it's a very Ingresse-specific situation that we're in right now that requires us to ask for it this way anyway, from my point of view)

Reply:
> A little bit, but we do have ingresse as a possible fraud_signal_type

oh wow that's very Ingresse-specific haha I'm going to add that in then

Reply:
>  The mechanism by which you identified the sale as fraudulent. Current accepted values are 'acquirer', other antifraud services such as 'horus', 'fraudnet', and 'sift', your company name ('ingresse'), or simply 'chargeback'.